interpreter/interpreter.go

```diff
@@ -136,7 +136,36 @@ func (itrp *Interpreter) doPackageClause(pkg *semantic.PackageClause) error {
 	return nil
 }
 
-func (itrp *Interpreter) doImport(dec *semantic.ImportDeclaration, scope values.Scope, importer Importer) error {
+type key int
+
+const packagesKey key = iota
+
+type Packages map[string]*Package
+
+func (p Packages) Inject(ctx context.Context) context.Context {
+	return context.WithValue(ctx, packagesKey, p)
+}
+
+func GetPackages(ctx context.Context) Packages {
```

Comment:
This is only used from inside this package, it would be good to change the name to `getPackages` to avoid exposing it.